src/Surfnet/StepupMiddleware/CommandHandlingBundle/Configuration/CommandHandler/InstitutionConfigurationCommandHandler.php

```diff
@@ -61,6 +62,27 @@ public function handleAddRaLocationCommand(AddRaLocationCommand $command)
             $institutionConfiguration = InstitutionConfiguration::create($institutionConfigurationId, $institution);
```

Comment:
it must always be created. It may not exist (if it exists we apparently made a mistake), the try catch should be removed.